conf/country-codes-en.properties

```diff
@@ -0,0 +1,249 @@
+AFG=Afghanistan
+ALB=Albania
+DZA=Algeria
+ASM=American Samoa
+AND=Andorra
+AGO=Angola
+AIA=Anguilla
+ATA=Antarctica
+ATG=Antigua and Barbuda
+ARG=Argentina
+ARM=Armenia
+ABW=Aruba
+AUS=Australia
+AUT=Austria
+AZE=Azerbaijan
+BHS=Bahamas
+BHR=Bahrain
+BGD=Bangladesh
+BRB=Barbados
+BLR=Belarus
+BEL=Belgium
+BLZ=Belize
+BEN=Benin
+BMU=Bermuda
+BTN=Bhutan
+BOL=Bolivia
+BES=Bonaire, Sint Eustatius and Saba
+BIH=Bosnia and Herzegovina
+BWA=Botswana
+BVT=Bouvet Island
+BRA=Brazil
+IOT=British Indian Ocean Territory
+UMI=United States Minor Outlying Islands
+VGB=Virgin Islands (British)
+VIR=Virgin Islands (U.S.)
+BRN=Brunei Darussalam
+BGR=Bulgaria
+BFA=Burkina Faso
+BDI=Burundi
+KHM=Cambodia
+CMR=Cameroon
+CAN=Canada
+CPV=Cabo Verde
+CYM=Cayman Islands
+CAF=Central African Republic
+TCD=Chad
+CHL=Chile
+CHN=China
+CXR=Christmas Island
+CCK=Cocos (Keeling) Islands
+COL=Colombia
+COM=Comoros
+COG=Congo
+COD=Congo (Democratic Republic of the)
+COK=Cook Islands
+CRI=Costa Rica
+HRV=Croatia
+CUB=Cuba
+CUW=Curaçao
+CYP=Cyprus
+CZE=Czech Republic
+DNK=Denmark
+DJI=Djibouti
+DMA=Dominica
+DOM=Dominican Republic
+ECU=Ecuador
+EGY=Egypt
+SLV=El Salvador
+GNQ=Equatorial Guinea
+ERI=Eritrea
+EST=Estonia
+ETH=Ethiopia
+FLK=Falkland Islands (Malvinas)
+FRO=Faroe Islands
+FJI=Fiji
+FIN=Finland
+FRA=France
+GUF=French Guiana
+PYF=French Polynesia
+ATF=French Southern Territories
+GAB=Gabon
+GMB=Gambia
+GEO=Georgia
+DEU=Germany
+GHA=Ghana
+GIB=Gibraltar
+GRC=Greece
+GRL=Greenland
+GRD=Grenada
+GLP=Guadeloupe
+GUM=Guam
+GTM=Guatemala
+GGY=Guernsey
+GIN=Guinea
+GNB=Guinea-Bissau
+GUY=Guyana
+HTI=Haiti
+HMD=Heard Island and McDonald Islands
+VAT=Holy See
+HND=Honduras
+HKG=Hong Kong
+HUN=Hungary
+ISL=Iceland
+IND=India
+IDN=Indonesia
+CIV=Côte d'Ivoire
```

Comment:
Can we check the style of apostrophe here please

Reply:
This is a good point Andy, currently it won't render it (if I recall how it works in play messages file) - we need to use "smart quotes" instead Mike https://smartquotesforsmartpeople.com/
I.e. replace `'` with `’`